packages/adminrouter/extra/src/README.md

```diff
@@ -264,28 +264,178 @@ it, but it does not seem to be justified. For example:
  * `lib/auth/open.lua` uses the `.init()` pattern in order to achive different
    behaviour depending on the flavour.
 
-## Service Endpoints
-
-Admin Router allows Marathon tasks to define custom service UI and HTTP endpoints, which are made available via `<dcos-cluster>/service/<service-name>`. This can be achieved by setting the following Marathon task labels:
+## Service Endpoint
+Admin Router offers the `/service` endpoint which enables users to easily
+access some of the tasks launched on DC/OS. AR internally pre-fetches data from
+Mesos and root Marathon and stores it in internal cache for later use when
+routing an incoming request.
 
+Services can be accessed both using namespace and name and by frameworkId in case of
+Mesos frameworks. For example - let's deploy a SchmetterlingDB framework into
+`/group1/group2/schmetterlingA`. It will be accessible using its ID (let's assume
```

Comment:
```
~ 273 Services can be accessed using both application ID and by framework ID in the
~ 274 case of Mesos frameworks. For example - let's deploy a SchmetterlingDB
~ 275 framework with framework ID equal to `/group1/group2/schmetterlingA`. It will
~ 276 be accessible using its ID (let's assume that Mesos assigned it a framework ID
+ 277 of `819aed93-4143-4291-8ced-5afb5c726803-0000`):
```